### ISSUE TITLE
user state implementation

### ISSUE THREAD

**Issue by LeeMooreImperas** [riscv-software-src/riscv-isa-sim#166]
Hi All,

Are the user level interrupt handling (N Extentsion) and user level trap handling implemented in spike, as per the privilege 1.10 specification ?

It would appear from tests we have written that they are not, and additionally we cannot see a method of enabling the N extension, eg 
isa=RV64GCN

Thx
Lee

**Comment by aswaterman** [comment]
You are correct. Also, N isn’t finalized yet (i.e. while we are pretty sure the design is solid, there is a slight chance it will be revised).
